ouroboros-consensus/ouroboros-consensus.cabal

```diff
@@ -664,3 +664,36 @@ benchmark mempool-bench
   -- https://github.com/Bodigrim/tasty-bench#comparison-against-baseline
   if impl(ghc >=8.6)
     ghc-options: -fproc-alignment=64
+
+benchmark ChainSync-client-bench
+  type:             exitcode-stdio-1.0
+  hs-source-dirs:   bench/ChainSync-client-bench
+  main-is:          Main.hs
+  other-modules:
+    Bench.Consensus.ChainSyncClient.Driver
+
+  build-depends:
+    , array
+    , base
+    , cardano-crypto-class
+    , containers
+    , contra-tracer
+    , ouroboros-consensus
+    , ouroboros-network-api
+    , ouroboros-network-protocols
+    , typed-protocols-examples
+    , unstable-consensus-testlib
+    , time
+
+  default-language: Haskell2010
+  ghc-options:
+    -Wall -Wcompat -Wincomplete-uni-patterns
+    -Wincomplete-record-updates -Wpartial-fields -Widentities
+    -Wredundant-constraints -Wmissing-export-lists -Wunused-packages
+    -Wno-unticked-promoted-constructors -rtsopts -with-rtsopts=-A32m
+
+  -- We use this option to avoid skewed results due to changes in cache-line
+  -- alignment. See
+  -- https://github.com/Bodigrim/tasty-bench#comparison-against-baseline
+  if impl(ghc >=8.6)
+    ghc-options: -fproc-alignment=64
```

Comment:
I think it is worth grouping this in a common stanza for benchmarks.

Reply:
Done